assets/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mayflower-theme",
+  "version": "1.0.0",
+  "description": "Mayflower theming assets, including color variables, fonts, icons and other imagery, along with componentized scss used to style Mayflower markup",
+  "main": "index.js",
```

Comment:
nit: I think we can get rid of `main` if `index.js` doesn't exist in this directory

Reply:
thanks good catch! It's missing some commits here

Reply:
Can I leave these in here. I think the rest of the PR #939 address updates this file and adds the entry points